### ISSUE TITLE
frontend: set password page

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#41]
## User story
1. As a { HR employee}
2. I want / need {to be able to set a password for my DIdentity- account}
3. So that {I can access DIdentity}

## Acceptance criteria
* page with two input fields (password and password confirmation) is shown plus a confirm-button
* tell the user the password rules: 8 elements. Must contain digits, letters, and special characters
* if input is equal, forward input to backend and wait for response of the backend. If everything successfull, show success-notification "Your password has been successfully set." 
* if input is not equal or doesn't match password rules, show error message and ask user to input passwords again.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
